frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -215,7 +217,7 @@ export function AccountsSummary({ accounts, devices }: TProps) {
                   totalBalancePerCoin={ balancePerCoin ? balancePerCoin[keystore.rootFingerprint] : undefined}
                   totalBalance={ accountsTotalBalance ? accountsTotalBalance[keystore.rootFingerprint] : undefined}
                   balances={balances}
-                  lightningNodeState={nodeState}
+                  lightningNodeState={lightningConfig.accounts[0].rootFingerprint === keystore.rootFingerprint ? nodeState : undefined}
```

Comment:
actually, I think all the nodeState so far are only used inside SummaryBalance component.

I think we could refactor and move all the lightning stuff into that component.